### PR TITLE
Add link compression tutorial

### PR DESCRIPTION
**Pull request by mDuo13**
- Add tutorial for enabling link compression as introduced in https://github.com/ripple/rippled/pull/3287
- Add mention of the tutorial to Capacity Planning document
- Move metadata to frontmatter for the two affected pages

### CODE REVIEW

**Review by gregtatcam — APPROVED** [review]
Looks good. One question. Does it make sense to mention that a subset of protocol messages is compressed? We only attempt to compress the following messages: protocol::mtMANIFESTS, protocol::mtENDPOINTS, protocol::mtTRANSACTION, protocol::mtGET_LEDGER, protocol::mtLEDGER_DATA, protocol::mtGET_OBJECTS, protocol::mtVALIDATORLIST. Also, if a message is not compressible, then we send uncompressed message.

File: content/tutorials/manage-the-rippled-server/configure-peering/enable-link-compression.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Enable Link Compression
+
+The `rippled` server can save bandwidth by compressing its [peer-to-peer communications](peer-protocol.html), at a cost of greater CPU usage. If you enable link compression, the server automatically compresses communications with peer servers that also have link compression enabled. The server uses uncompressed communications with peers who do not have have link compression enabled or do not support link compression. [New in: rippled 1.6.0][]
+
+## Steps
+
+To enable link compression on your server, complete the following steps:
+
+### 1. Edit your `rippled`'s config file.
+
```
Comment (ddibble-rip) [inline content/tutorials/manage-the-rippled-server/configure-peering/enable-link-compression.md:18]: Why not use the following? Seems weird to add the possessive to `rippled`, and using "server" is consistent with the other docs.
  
Edit your `rippled` server's config file.

File: content/tutorials/manage-the-rippled-server/configure-peering/enable-link-compression.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+funnel: Docs
+doc_type: Tutorials
+category: Manage the rippled Server
+subcategory: Configure Peering
+blurb: Save bandwidth by compressing peer-to-peer communications.
+---
+# Enable Link Compression
+
+The `rippled` server can save bandwidth by compressing its [peer-to-peer communications](peer-protocol.html), at a cost of greater CPU usage. If you enable link compression, the server automatically compresses communications with peer servers that also have link compression enabled. The server uses uncompressed communications with peers who do not have have link compression enabled or do not support link compression. [New in: rippled 1.6.0][]
+
```
Comment (ddibble-rip) [inline content/tutorials/manage-the-rippled-server/configure-peering/enable-link-compression.md:12]: Was wondering how much CPU usage would be saved by this. Makes sense to compress bandwidth if you knew this setting is the norm for most other hosts.

Anyway to recast the last sentence? There's also a repeated "have" in the sentence:

The server uses uncompressed communications with peers who do not **have have** link compression enabled or do not support link compression.

Something like this, maybe?

Peer-to-peer link compression works only with peer servers that support link compression and have link compression enabled. In the absence of either, your server defaults to uncompressed communications with that peer.

**Review by ddibble-rip — COMMENTED** [review]
Hi Rome,  
Just a couple of comments around clarity, and repeated word within a paragraph. Otherwise,  looks great.

File: content/tutorials/manage-the-rippled-server/configure-peering/enable-link-compression.md (hunk 2 above)
Comment (mDuo13) [reply content/tutorials/manage-the-rippled-server/configure-peering/enable-link-compression.md:12]: Was thinking about it and I decided the best way to rephrase the sentence was to remove it.

It's implied by the previous sentence and awkwardly long pretty much no matter what, so I think in reality it's more likely to confuse people than it is to help.

**Pull request merged**